speed-dreams/source-2.2.3/src/drivers/replaydriver/Driver.cpp

```diff
@@ -25,16 +25,18 @@ Driver::Driver(int p_index, const char* p_name)
 /// @param p_situation The current race situation
 void Driver::InitTrack(tTrack* p_track, void* p_carHandle, void** p_carParmHandle, tSituation* p_situation)
 {
-    std::experimental::filesystem::path sdaFolder;
+    using std::experimental::filesystem::path;
```

Comment:
this is fine, but maybe consider shortening the namespace E.G. 
```
namespace filesystem = std::experimental::filesystem
```

Reply:
Will that cause issues if we ever update our c++ version?

Reply:
Well yes but then its easier to fix as we can just change std::expreimental::filesystem to std::filesystem